src/components/addBook.jsx

```diff
@@ -1,27 +1,45 @@
+import { nanoid } from '@reduxjs/toolkit';
 import { useDispatch } from 'react-redux';
-import { addBook } from '../redux/books/booksSlice';
+import { addBook, fetchBooks } from '../redux/books/booksSlice';
 import './addBook.scss';
 
 const AddBook = () => {
   const dispatch = useDispatch();
 
-  const handleFormSubmit = (event) => {
+  const handleFormSubmit = async (event) => {
+    dispatch(fetchBooks());
     event.preventDefault();
     const title = event.target.bookTitle.value;
     const author = event.target.author.value;
-    dispatch(addBook({ title, author }));
+    const category = event.target[2].options[event.target[2].selectedIndex].text;
+    const itemId = nanoid();
+
+    dispatch(addBook({
+      title,
+      author,
+      item_id: itemId,
+      category,
+    }));
     event.target.reset();
+    console.log(category);
   };
 
   return (
     <div className="add-book">
       <h4>ADD NEW BOOK</h4>
       <form className="row g-3 mt-2" onSubmit={handleFormSubmit}>
-        <div className="col-6">
-          <input type="text" className="form-control" id="book-title" name="bookTitle" placeholder="Book Title" />
-        </div>
         <div className="col-4">
-          <input type="text" className="form-control" id="author" name="author" placeholder="Author" />
+          <input type="text" className="form-control" id="book-title" name="bookTitle" placeholder="Book Title" required />
+        </div>
+        <div className="col-3">
+          <input type="text" className="form-control" id="author" name="author" placeholder="Author" required />
+        </div>
+        <div className="col-3">
+          <select className="form-control" name="category">
+            <option value="Action">Action</option>
+            <option value="Science Fiction">Science Fiction</option>
+            <option value="Economy">Economy</option>
+          </select>
         </div>
         <div className="col-2">
           <button type="submit" className="btn btn-primary mb-3">ADD BOOK</button>
```

Comment:
- Kindly make sure there is a gap margin bottom below the form as per the Zeplin design:
Your design:
![Screenshot from 2023-04-15 23-43-55](https://user-images.githubusercontent.com/97834160/232254352-dc80728c-82bf-4f35-87a8-7a6b73b72c44.png)
Zeplin:
![Screenshot from 2023-04-15 23-44-50](https://user-images.githubusercontent.com/97834160/232254377-fa8c9c9a-e348-43cc-a304-00a48662695c.png)